docs/reference/ml/apis/put-dfanalytics.asciidoc

```diff
@@ -0,0 +1,105 @@
+[role="xpack"]
+[testenv="platinum"]
+[[put-dfanalytics]]
+=== Create {dfanalytics-jobs} API
+[subs="attributes"]
+++++
+<titleabbrev>Create {dfanalytics-jobs}</titleabbrev>
+++++
+
+beta[]
+
+Instantiates a {dfanalytics-job}.
+
+[[ml-put-dfanalytics-request]]
+==== {api-request-title}
+
+`PUT _ml/data_frame/analytics/<data_frame_analytics_id>`
+
+[[ml-put-dfanalytics-prereq]]
+==== {api-prereq-title}
+
+* You must have `machine_learning_admin` built-in role to use this API. You must 
+also have `read` and `view_index_metadata` privileges on the source index and 
+`read`, `create_index`, and `index` privileges on the destination index. For 
+more information, see {stack-ov}/security-privileges.html[Security privileges] 
+and {stack-ov}/built-in-roles.html[Built-in roles].
+
+[[ml-put-dfanalytics-path-params]]
+==== {api-path-parms-title}
+
+`<data_frame_analytics_id>` (Required)::
+  (string) A numerical character string that uniquely identifies the 
+  {dfanalytics-job}. This identifier can contain lowercase alphanumeric characters 
+  (a-z and 0-9), hyphens, and underscores. It must start and end with alphanumeric 
+  characters.
+
+[[ml-put-dfanalytics-request-body]]
+==== {api-request-body-title}
+
+`source` (Required)::
+  (object) The source configuration, consisting of `index` and optionally a 
+  `query`.
+  
+`dest` (Required)::
+  (object) The destination configuration, consisting of `index` and optionally 
```

Comment:
I am not sure where the right place for this would be but there are some important things to explain about the destination index. I'll note them here and we can decide where and how they should be documented.

The destination index will be automatically created if it does not already exist. When this is the case we copy the `index.number_of_shards` and `index.number_of_replicas` settings only. When `source.index` matches multiple indices, we set those settings to the max values found in the source indices. We also attempt to copy the mappings from the source indices over to the destination index that is about to be created. However, if the mappings for any of the fields are not the same among the source indices, we will fail with an error message.

If the destination index already exists, then it will be used as-is. This allows users to set-up the destination index  in advance with custom settings and mappings.

Reply:
I added it to the `dest` description as it seemed the most appropriate to me.

Reply:
I think that kind of detail can go into a "Description" section for the API. Otherwise if you leave them where they are, I think you need to add continuation characters per https://asciidoctor.org/docs/asciidoc-writers-guide/#list-continuation

Otherwise they're showing up like this:
![image](https://user-images.githubusercontent.com/26471269/60632292-b30d1380-9db9-11e9-9474-9e803151e9cb.png)

Reply:
I created a Description section and relocated the explanation there with an introductory sentence.